src/services/NotificationService.ts

```diff
@@ -21,30 +18,23 @@ class NotificationService {
    * Sending email with the certificate according to the given params
    * @param params - personalization details,email and certificate
    * @param emails - emails to send to
-   * @param emailType - email receiver type
+   * @param activityId - activityId for logging purposes
    */
-  public async sendNotification(params: any, emails: string[], emailType: string, activityId: string): Promise<any[]> {
+  public async sendNotification(params: any, emails: string[], activityId: string) {
     const templateId: string = await this.config.getTemplateIdFromEV();
     const emailDetails = {
       personalisation: params,
     };
-    const sendEmailPromise = [];
 
     for (const email of emails) {
-      const sendEmail = this.notifyClient.sendEmail(templateId, email, emailDetails).then((response: any) => response.data);
-      sendEmailPromise.push(sendEmail);
+      try {
+        await this.notifyClient.sendEmail(templateId, email, emailDetails);
+        console.log(`report successfully sent email to ${email} for PNumber ${params.testStationPNumber} with activity ${activityId}.`);
```

Comment:
Thinking about this, not sure if we want to be logging email as their could be a testers email address